compiler/rustc_mir_transform/src/remove_noop_landing_pads.rs

```diff
@@ -25,7 +26,24 @@ impl<'tcx> crate::MirPass<'tcx> for RemoveNoopLandingPads {
             .iter_enumerated()
             .any(|(_bb, block)| matches!(block.terminator().kind, TerminatorKind::UnwindResume));
         if !has_resume {
-            debug!("remove_noop_landing_pads: no resume block in MIR");
+            debug!("no resume block in MIR");
+            return;
+        }
+
+        let mut nop_landing_pads = DenseBitSet::new_empty(body.basic_blocks.len());
+
+        // This is a post-order traversal, so that if A post-dominates B
+        // then A will be visited before B.
+        for &bb in body.basic_blocks.reverse_postorder().iter().rev() {
```

Comment:
why `body.basic_blocks.reverse_postorder().iter().rev()` instead** of `traversal::postorder(body)`?

r=me otherwise